docs/reference/commandline/daemon.md

```diff
@@ -67,6 +67,7 @@ weight = -1
       --tlsverify                            Use TLS and verify the remote
       --userns-remap="default"               Enable user namespace remapping
       --userland-proxy=true                  Use userland proxy for loopback traffic
+      --userland-proxy-bin=""                Use this userland proxy binary, if userland-proxy is set
```

Comment:
Is this just the binary name, or the full path? If we expect a full path, should we call it `--userland-proxy-path` ?

On the other hand, the external credential stores use just the *name*, and requires the binary to be in `$PATH` (see https://github.com/docker/docker/pull/20107)